executor/analyze.go

```diff
@@ -288,20 +306,22 @@ func (e *AnalyzeIndexExec) buildStats() (hist *statistics.Histogram, cms *statis
 		if l := nullHist.Len(); l > 0 {
 			hist.NullCount = nullHist.Buckets[l-1].Count
 		}
+		e.job.Update(hist.NullCount)
```

Comment:
We don't need to add `hist.NullCount` here since `buildStatsFromResult` has already added it?